docs/source/security/authentication.mdx

```diff
@@ -263,13 +289,12 @@ context: ({ req }) => {
 },
 ```
 
-```js
-// src/models/user.js
+```ts title="src/models/user.ts"
 export const generateUserModel = ({ req }) => ({
- getAll: () => {
-   return fetch('http://myurl.com/users', { headers: req.headers });
- },
+  getAll: () => {
+    return fetch('http://myurl.com/users', { headers: req.headers });
```

Comment:
this seems like an intense default; i'd suggest more explicitly passing a single api-key header through instead of forwarding every header?